app/halgo/Main.hs

```diff
@@ -403,10 +427,10 @@ cmdNodeSend json url = do
 -- | Get txn status
 cmdNodeTxnStatus :: MonadSubcommand m => Text -> NodeUrl -> m ()
 cmdNodeTxnStatus txId url = withNode url $ \(_, api) ->
-  N.transactionStatus <$> Api._transactionsPending api txId >>= \case
-    N.Waiting -> putTextLn $ "Waiting"
+  Api._transactionsPending api txId >>= (\case
+    N.Waiting -> putTextLn "Waiting"
     N.KickedOut reason -> die $ "Kicked out: "+|reason|+""
-    N.Confirmed r -> putTextLn . pretty $ "Confirmed in round " <> show r
+    N.Confirmed r -> putTextLn . pretty $ "Confirmed in round " <> show r) . N.transactionStatus
```

Comment:
Is it just a cosmetics change? If yes, I would avoid it because it's subjective and irrelevant to this PR. But maybe I am missing something other than cosmetics.

Reply:
mm.. yes, it's cosmetic one
I'll return it back